modules/migration-mtc-release-notes-1-7-16.adoc

```diff
@@ -0,0 +1,77 @@
+
+A flaw was found in the `net/mail` Golang standard library package, which impacts earlier versions of {mtc-short}. The `ParseAddressList` function incorrectly handles comments, text in parentheses, and display names. As this is a misalignment with conforming address parsers, it can result in different trust decisions being made by programs using different parsers.
+
+To resolve this issue, upgrade to {mtc-short} 1.7.16.
+
+For more details, see link:https://access.redhat.com/security/cve/cve-2024-24784[CVE-2024-24784].
+
+.CVE-2024-24785: Golang: `html/template`: Errors returned from `MarshalJSON` methods may break template escaping
+
+A flaw was found in the `html/template` Golang standard library package, which impacts earlier versions of {mtc-short}. If errors returned from `MarshalJSON` methods contain user-controlled data, they could be used to break the contextual auto-escaping behavior of the `html/template` package, allowing subsequent actions to inject unexpected content into templates.
+
+To resolve this issue, upgrade to {mtc-short} 1.7.16.
+
+For more details, see link:https://access.redhat.com/security/cve/cve-2024-24785[CVE-2024-24785].
+
+.CVE-2024-29180: `webpack-dev-middleware`: Lack of URL validation may lead to file leak
+
+A flaw was found in the `webpack-dev-middleware package`, which impacts earlier versions of {mtc-short}. This flaw fails to validate the supplied URL address sufficiently before returning local files, which could allow an attacker to craft URLs to return arbitrary local files from the developer's machine.
+
+To resolve this issue, upgrade to {mtc-short} 1.7.16.
+
+For more details, see link:https://access.redhat.com/security/cve/cve-2024-29180[CVE-2024-29180].
+
+.CVE-2024-30255: `envoy`: HTTP/2 CPU exhaustion due to CONTINUATION frame flood
+
+A flaw was found in how the `envoy` proxy implements the HTTP/2 codec, which impacts earlier versions of {mtc-short}. There are insufficient limitations placed on the number of `CONTINUATION` frames that can be sent within a single stream, even after exceeding the header map limits of `envoy`. This flaw could allow an unauthenticated remote attacker to send packets to vulnerable servers. These packets could use up compute resources to cause a denial of service (DoS).
+
+To resolve this issue, upgrade to {mtc-short} 1.7.16.
+
+For more details, see link:https://access.redhat.com/security/cve/cve-2024-30255[CVE-2024-30255].
+
+
+[id="known-issues-1-7-16_{context}"]
+== Known issues
+
+This release has the following known issues:
+
+.Direct Volume Migration is failing as the Rsync pod on the source cluster goes into an `Error` state
+
+On migrating any application with a Persistent Volume Claim (PVC), the `Stage` migration operation succeeds with warnings, and the Direct Volume Migration (DVM) fails with the `rsync` pod on the source namespace moving into an `error` state. link:https://bugzilla.redhat.com/show_bug.cgi?id=2256141[BZ#2256141]
```

Comment:
```suggestion
On migrating any application with a Persistent Volume Claim (PVC), the `Stage` migration operation succeeds with warnings, but the Direct Volume Migration (DVM) fails with the `rsync` pod on the source namespace moving into an `error` state. link:https://bugzilla.redhat.com/show_bug.cgi?id=2256141[(BZ#2256141)]
```
In the previous "Known issues" sections, the BZ links are bolded. For consistency, you might consider doing that here as well.